72_config_and_simplify_tls.md

```diff
@@ -0,0 +1,804 @@
+allows to set different private keys and certificates to be used when accessing
+each different component. In the new configuration format, all services are
+accessed using the same private key (`client_key`) and certificate
+(`client_cert`).
+
+To allow the components to authenticate the tenant, it is necessary to add the
+tenant CA certificate to the `trusted_client_ca` list in each component
+configuration.
+
+Also the option to use password protected private keys is no longer supported,
+allowing the `private_key_pw`, `agent_mtls_private_key_pw`, and
+`registrar_private_key_pw` to be removed.
+
+| Old option                 | New Option          | Notes |
+| -------------------------- | ------------------- | ----- |
+| `tls_dir`                  | `tls_dir`           | |
+| `ca_cert`                  | `trusted_server_ca` | Add the verifier CA certificate to `trusted_server_ca`|
+| `my_cert`                  | `client_cert`       | |
+| `private_key`              | `client_key`        | |
+| `private_key_pw`           | (None, removed)     | Removed support for password protected private keys |
+| `check_client_cert`        | (None, removed)     | Use `all` keyword in `trusted_client_ca`|
+| `agent_mtls_cert_enabled`  | `enable_agent_mtls` | |
+| `agent_mtls_cert`          | (None, removed)     | Add the CA of the agent's `server_cert` to `trusted_server_ca`
+| `agent_mtls_private_key`   | `client_key`        | |
+| `agent_mtls_private_key_pw`| (None, removed)     | Removed support for password protected private keys |
+| `registrar_tls_dir`        | (None, removed)     | Use `tls_dir` |
+| `registrar_ca_cert`        | (None, removed)     | Add the CA of the registrar's `server_cert` to `trusted_server_ca`|
+| `registrar_my_cert`        | `client_cert`       | |
+| `registrar_private_key`    | `client_key`        | |
+| `registrar_private_key_pw` | (None, removed)     | Removed support for password protected private keys |
+| `check_client_cert`        | (None, removed)     | Use `all` keyword in `trusted_client_ca`|
+
+### Registrar
+
+For the registrar, the options are renamed to follow the new naming scheme and
+some options are removed since there is no need to set multiple server private
+keys and certificates.
+
+The registrar does not access other services as a client, thus it does not have
+the `client_key`, `client_cert`, and `trusted_server_ca` options.
+
+Also the option to use password protected private keys is no longer supported,
+allowing the `private_key_pw` and `registrar_private_key_pw` to be removed.
+
+The following table summarizes the changes for the registrar TLS configuration.
+
+| Old option                 | New Option          | Notes |
+| -------------------------- | ------------------- | ----- |
+| `registrar_tls_port`        | `tls_port`         | |
+| `tls_dir`                  | `tls_dir`           | |
+| `ca_cert`                  | `trusted_client_ca` | Add the clients CA certificates to `trusted_client_ca`|
```

Comment:
```suggestion
| `ca_cert`                  | `trusted_client_ca` | Add the tenant CA certificates to `trusted_client_ca`|
```

Reply:
Thank you, I applied the suggested change.